### DISCUSSION TITLE
Discord unban

### DISCUSSION THREAD

**Discussion by Kr3yx** [pterodactyl/panel#5106] [Support] [answered]
Hello I would like to be unbanned from your Pterodactyl discord, I was just banned why I don't know. I just asked if anyone knows any addons or ways to offer free gameservers to other people. I was then directed to the panel Helping channel and posted the same question there and I was banned from the discord. I just want to do others a favor :( I would like to know why I was banned and if it was really wrong or an erroneous ban. :(

Discord Name:
zkreyx

**Comment by rowansc1** [comment, marked as answer]
Your username cannot be located.
